### ISSUE TITLE
1.4.1 throws java.lang.NoSuchMethodError ConcurrentLinkedHashMap$Builder.maximumWeightedCapacity

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vadbch/orika#88]
```
What steps will reproduce the problem?
1.Building and running Orika with fresh updates from the Maven Repo
2.Using a custom converter.

What version of the product are you using? On what operating system?
1.4.1

Please provide any additional information below.

With the 1.4.1 release Orika moved to use ConcurrentLinkedHashMap but the 
method ConcurrentLinkedHashMap$Builder.maximumWeightedCapacity has been updated 
as part of the 5/2012 release of ConcurrentLinkedHashMap to take a long, not an 
int.
http://code.google.com/p/concurrentlinkedhashmap/wiki/Changelog

In ma.glasnost.orika.converter.DefaultConverterFactory an int is used (line 
38). Please change this to a long.


Thanks!


```

Original issue reported on code.google.com by `mwiel...@gmail.com` on 11 Mar 2013 at 8:40

**Comment by GoogleCodeExporter** [comment]
```
Unfortunately, Orika targets jdk 1.5 as minimum, and AFAIK, the latest 1.5 
version available is the 1.2 version we've included (which has the int 
signature); it won't compile if we try passing a long into that signature 
without casting it : )
This problem is because you're using a newer version of this class (I'm 
guessing), and they've (accidentally?) broken back-compatibility from the 1.2 
to 1.3 versions.
I've included a reflection-based workaround in the lastest 
1.4.2-SNAPSHOT...please check it out and confirm that it fixes the issue for 
you.
```

Original comment by `matt.deb...@gmail.com` on 6 May 2013 at 6:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.deb...@gmail.com` on 6 May 2013 at 6:20
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.4.2 release
```

Original comment by `matt.deb...@gmail.com` on 20 May 2013 at 6:40
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks Matt. I just tested it and there is one more place where it's 
complaining.

Line 63 of ma.glasnost.orika.impl.MapperFacadeImpl
```

Original comment by `mwiel...@gmail.com` on 21 May 2013 at 6:37
